From c7e63da760b639bb8205015e9809ec0f4dbe1ac1 Mon Sep 17 00:00:00 2001
From: Vishal Sagar <vishal.sagar@xilinx.com>
Date: Sun, 15 Mar 2020 23:07:05 -0700
Subject: [PATCH 1274/1852] dt: bindings: media: xilinx: sdirxss : Add GT reset
 gpio handle

commit 778f2855812940fcaea0f170aeb031803a627500 from
https://github.com/Xilinx/linux-xlnx.git

Add optional property to reset the compact GT using a GPIO pin connected
to fmc init done pin. This signal is active low.

Signed-off-by: Vishal Sagar <vishal.sagar@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 .../devicetree/bindings/media/xilinx/xlnx,sdirxss.txt       | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/Documentation/devicetree/bindings/media/xilinx/xlnx,sdirxss.txt b/Documentation/devicetree/bindings/media/xilinx/xlnx,sdirxss.txt
index 1e38b7d5c5a9..114e228ae180 100644
--- a/Documentation/devicetree/bindings/media/xilinx/xlnx,sdirxss.txt
+++ b/Documentation/devicetree/bindings/media/xilinx/xlnx,sdirxss.txt
@@ -35,6 +35,11 @@ Required properties:
   video.txt. Please note that the video format is fixed to either YUV422 or YUV420
   and the video-width is 10.
 
+Optional properties:
+
+- reset_gt-gpios: contains GPIO reset phandle for FMC init done pin in GT.
+  This pin is active low.
+
 Example:
 		v_smpte_uhdsdi_rx_ss: v_smpte_uhdsdi_rx_ss@80000000 {
 			compatible = "xlnx,v-smpte-uhdsdi-rx-ss";
@@ -47,6 +52,7 @@ Example:
 			xlnx,line-rate = "12G_SDI_8DS";
 			clocks = <&clk_1>, <&si570_1>, <&clk_2>;
 			clock-names = "s_axi_aclk", "sdi_rx_clk", "video_out_clk";
+			reset_gt-gpios = <&axi_gpio_0 0 0 GPIO_ACTIVE_LOW>;
 
 			ports {
 				#address-cells = <1>;
-- 
2.31.1

